spec/controllers/kudos_controller_spec.rb

```diff
@@ -6,38 +6,66 @@
   include LoginMacros
   include RedirectExpectationHelper
 
-  describe "create" do
+  describe "POST #create" do
     context "when work is public" do
       let(:work) { create(:posted_work) }
+      let(:referer) { work_path(work) }
+      before { request.headers["HTTP_REFERER"] = referer }
```

Comment:
Previously we had `referer = work_chapter_path(work, work.first_chapter)` in one case, but for the purpose of this test it doesn't matter what the referer is, only that it is present. So I pulled it out here.